lambda_src/log.py

```diff
@@ -47,3 +48,35 @@ def format_trace(e: Exception) -> str:
 
     pid = getpid()
     return f'[{pid}] {a}'
+
+
+def setup_logger(logger_name: str, level: int = logging.INFO) -> Logger:
+    """Sets up the logger object.
+
+    Args:
+        logger_name (str): Name to use to retreive the logger instance.
+        level (int, optional): Level of logging. Defaults to logging.INFO.
+        stdout (bool, optional): Bool whether to print to stdout or not. Defaults to False.
+
+    Returns:
+        Logger: Returns the logger object.
+    """
+    l = logging.getLogger(logger_name)
+    l.setLevel(level)
+    return l
+
+
+def get_loggers() -> Tuple[Logger, Logger, Logger]:
```

Comment:
do we always need all three loggers? would it make sense to have the calling code just get the loggers they need instead of calling `get_loggers` which calls `getLogger` three times, of which only one might be used?

Reply:
actually, u're right. We don't. let me fix.